models/src/main/scala/coop/rchain/casper/PrettyPrinter.scala

```diff
@@ -63,4 +68,11 @@ object PrettyPrinter {
 
   def buildString(b: Bond): String =
     s"${buildStringNoLimit(b.validator)}: ${b.stake.toString}"
+
+  def buildString(hashes: List[BlockHash]): String =
+    s"[${hashes
+      .foldLeft("") { (str, dep) =>
+        s"$str ${PrettyPrinter.buildString(dep)}"
+      }
+      .trim}]"
```

Comment:
With `mkString` method it's easier to join strings. Similar to this code.
https://github.com/rchain/rchain/blob/8489c6ca0f2ff8c3df1ebb8a63ca717c1a6cb07a/node/src/main/scala/coop/rchain/node/web/ReportingRoutes.scala#L28-L29